src/main/proto/wfa/measurement/reporting/v2alpha/metric.proto

```diff
@@ -128,47 +128,66 @@ message MetricSpec {
   VidSamplingInterval vid_sampling_interval = 5;
 }
 
+// Statistics of a scalar value
+message UnivariateStatistics {
+  // The standard deviation of the associated scalar variable value.
+  //
+  // Can be used to find different confidence intervals.
+  double standard_deviation = 1;
+}
+
 // The calculation result of a metric
 message MetricResult {
   // Reach result format.
-  //
-  // TODO(@riemanli): Add error bar.
   message ReachResult {
     // Reach value.
     int64 value = 1;
+    // Statistics of the reach value above
+    UnivariateStatistics statistics = 2;
```

Comment:
Similar to the last comment, the variable name "statistics" might be a little bit confusing.